src/python/platforms/android/settings.py

```diff
@@ -194,3 +228,15 @@ def set_database_setting(database_path, table, key, value):
   sql_command_string = ('"UPDATE %s SET value=\'%s\' WHERE name=\'%s\'"') % (
       table, str(value), key)
   adb.run_shell_command(['sqlite3', database_path, sql_command_string])
+
+
+def get_repo_prop_path():
+  """Download repo.prop and return path of it on local machine."""
+  symbols_directory = environment.get_value('SYMBOLS_DIR')
```

Comment:
Lets not use the same directory as userspace symbols, as otherwise we will constantly keep deleting symbols dir content, first with userspace symbols, then delete, then have kernel symbols, then delete, etc.
kernel_symbols_directory = os.path.join(symbols_directory, 'kernel')

Reply:
I added it as a new constant but yes